async-nats/src/lib.rs

```diff
@@ -1354,7 +1354,7 @@ pub struct ConnectInfo {
 }
 
 /// Protocol version used by the client.
-#[derive(Serialize_repr, Deserialize_repr, PartialEq, Eq, Debug, Clone, Copy)]
+#[derive(Serialize_repr, Deserialize_repr, PartialEq, Eq, Hash, Debug, Clone, Copy)]
```

Comment:
What is the use case for adding hash here?

Reply:
In order to store the `ConnectInfo`s in a hashmap, this needs to be `Hash` :)